docs/reference/content/getting-started/quick-tour-primer.md

```diff
@@ -0,0 +1,95 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Quick Tour Primer"
+[menu.main]
+  parent = "Getting Started"
+  identifier = "Primer"
+  weight = 20
+  pre = "<i class='fa'></i>"
++++
+
+# Quick Tour Primer
```

Comment:
This is inline with the RX and Reactive Stream drivers and is used to provide background to the library before using it.